Source/Scene/Cesium3DTile.js

```diff
@@ -1054,10 +1055,27 @@ define([
      * @private
      */
     Cesium3DTile.prototype.update = function(tileset, frameState) {
+        // Compute and compare ClippingPlanes state:
+        // - enabled-ness - are clipping planes enabled? is this tile clipped?
+        // - clipping plane count
+        // - clipping function (union v. intersection)
+        var clippingPlanes = tileset.clippingPlanes;
+        var currentClippingPlanesState = 0;
+        if (defined(clippingPlanes) && this._isClipped && clippingPlanes.enabled) {
+            currentClippingPlanesState = clippingPlanes.clippingPlanesState();
+        }
+        // If clippingPlaneState for this tile changed, mark content as clippingPlanesDirty
+        if (currentClippingPlanesState !== this._clippingPlanesState) {
+            this._clippingPlanesState = currentClippingPlanesState;
+            this._content.clippingPlanesDirty = true;
+        }
```

Comment:
Maybe add the clipping planes code to a separate function `updateClippingPlanes`? Just to help keep `update` super simple. Though `this._content.clippingPlanesDirty = false; // reset after content update` will have to stay here.